### ISSUE TITLE
Redundant mdtool steps being generated.

### ISSUE THREAD

**Issue by carlossless** [bitrise-io/steps-xamarin-test-cloud-for-android#16]
Hi, I'm having problems with my build not going through because of VM space constraints. However I noticed some redundant build steps being generated by this step:

```
["\"/Applications/Xamarin Studio.app/Contents/MacOS/mdtool\"", "build", "\"-c:UITest|iPhone\"", "\"Jobz.sln\"", "\"-p:Jobz.iOS\""]
...
["\"/Applications/Xamarin Studio.app/Contents/MacOS/mdtool\"", "archive", "\"-c:UITest|iPhone\"", "\"Jobz.sln\"", "\"-p:Jobz.iOS\""]
...
["\"/Applications/Xamarin Studio.app/Contents/MacOS/mdtool\"", "build", "\"-c:UITest|iPhone\"", "\"Jobz.sln\"", "\"-p:Jobz.iOS\""]
...
["\"/Applications/Xamarin Studio.app/Contents/MacOS/mdtool\"", "archive", "\"-c:UITest|iPhone\"", "\"Jobz.sln\"", "\"-p:Jobz.iOS\""]
...
["\"/Applications/Xamarin Studio.app/Contents/MacOS/mdtool\"", "build", "\"-c:Debug\"", "\"Jobz.sln\"", "\"-p:Jobz.UITests\""]
```

**Comment by viktorbenei** [comment]
Thanks for reporting, we'll try to see if we can figure out a way to reduce the storage consumption of this step.

**Comment by godrei** [comment]
Hi @carlossless ,
the issue is fixed in version [2.0.0](https://github.com/bitrise-steplib/steps-xamarin-test-cloud-for-android/releases/tag/2.0.0), available in the steplib.

**Comment by viktorbenei** [comment]
😱👍
